masterlist.yaml

```
@@ -7779,3 +7779,19 @@ plugins:
       - link: 'https://www.nexusmods.com/fallout4/mods/68889/'
         name: 'Extra Icons for FIS'
     after: [ 'FIS-Naming-Weap-Armo-EN.esp' ]
+
+  - name: 'Chameleon No Sound.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/1886'
```

Link is wrong, also missing a few things:

```suggestion
      - link: 'https://www.nexusmods.com/fallout4/mods/26648/'
        name: 'No Sound Chameleon cloak'
```